Source/WTF/wtf/text/StringCommon.h

```diff
@@ -597,10 +597,8 @@ ALWAYS_INLINE const uint8_t* find8(const uint8_t* pointer, uint8_t character, si
     return static_cast<const uint8_t*>(memchr(pointer + index, character, length - index));
 }
 
-WTF_EXPORT_PRIVATE const uint16_t* find16AlignedImpl(const uint16_t* pointer, uint16_t character, size_t length);
-
-#if CPU(ARM64)
-ALWAYS_INLINE const uint16_t* find16(const uint16_t* pointer, uint16_t character, size_t length)
+template<typename UnsignedType>
+ALWAYS_INLINE const UnsignedType* findImpl(const UnsignedType* pointer, UnsignedType character, size_t length)
 {
     // We take `size_t` length instead of `unsigned` because,
     // 1. It is aligned to memchr.
```

Comment:
The `// 2. It allows us to use find16 for 4GB~ vectors, which can be used in JSC ArrayBuffer (4GB wasm memory).` comment below this is stale, and should be updated.  "find16" ==> "findImpl".

Reply:
Removed.